test_start_page.py

```diff
@@ -18,21 +28,13 @@ def test_empty_email_alert(self):
             Clear the email field
             Check the email-field alert message
         """
-        driver = webdriver.Chrome(r"C:\Users\andrii.lazaryshyn\PycharmProjects\AQA_Selenium\chromedriver.exe")
-        driver.get("https://qa-complex-app-for-testing.herokuapp.com/")
-
-        log.info('Starting test - check email field alert')
-        email = driver.find_element(by=By.XPATH, value='.//*[@name="email" and @placeholder="you@example.com"]')
-        email.send_keys('villy@wonka.com')
-        sleep(2)
-        email.clear()
-        sleep(5)
-        alert_message = driver.find_element(by=By.XPATH, value='.//*[text()="You must provide a valid email address."]')
-        assert alert_message.is_displayed()
-        log.info('Test passed. Alert is displayed')
-        driver.close()
+        start_page.fill_and_clear(xpath=xpath, value=value)
```

Comment:
1. Why are you doing fill_and_clear ? Just clear wouldn't be enough ?
2. Please do not use these methods directly in tests. You have to reuse sign in/up methods with required values

Reply:
1. No it would not - I will not see the alert.
2. Where did you found those code? I do not have such in test_start_page file O_O